gensrc/thrift/StatusCode.thrift

```diff
@@ -82,6 +82,9 @@ enum TStatusCode {
     DUPLICATE_RPC_INVOCATION        = 47,
     GLOBAL_DICT_ERROR = 48,
 
-    UNKNOWN = 50
+    UNKNOWN = 50,
+
+    TRANSACTION_NOT_EXISTS = 51,
```

Comment:
It's better to use TXN. Because the txn and transaction are all OK.
So we misused the two words. But the log is difficult to remember.